### DISCUSSION TITLE
什么时候更新一下nuget上的版本啊

### DISCUSSION THREAD

**Discussion by lantongxue** [HandyOrg/HandyControl#1154] [General]
nuget上还是3.3.0，更新是4个月前，啥时候更新一下子呗。

**Comment by GF-Huang** [comment]
这个问题确实挺无奈的，我觉得每天构建一个 beta 版本传 NuGet 挺好。@NaBian

**Comment by KleinPan** [comment]
还没更新。。。

**Comment by NaBian** [comment]
其实原本有日构建版本的，但是现在 myget 好像不好用了，可能是空间不够，或者其它原因吧，具体不想细查了。
新版本预计在 11 月发布。

**Comment by NaBian** [comment]
done
